.github/actions/get-version/action.yml

```diff
@@ -18,10 +18,7 @@ runs:
       run: |
         IS_RELEASE="${{ inputs.is_release }}"
         echo "IS_RELEASE $IS_RELEASE"
-        if [[ "$IS_RELEASE" == "false" ]] ;then
-          export CI_VERSION=true
-        fi
-        version=$(./gradlew radixCiVersion | grep radixdlt-version | cut -d: -f2)
+        version=$(./gradlew getRadixVersion | grep radixdlt-version | cut -d: -f2)
```

Comment:
I wonder if we should just make it `printRadixVersion` and get rid of the `radixdlt-version:`prefix?

Reply:
I imagine we need to pass environment variables to this script? Similar to `setup_version_properties`?

Reply:
But also - I wonder if we actually need this action at all any more?  Happy to keep it or remove it, depending on what's most convenient.

Reply:
Will investigate the action. I just went for a dumb like-for-like replacement and didn't look too deeply into what the actions did/

Reply:
I kept the action, but removed `printRadixVersion` from the gradle file and get the version from the `setup-version-properties` action